save-preprocessor/src/main/kotlin/com/saveourtool/save/preprocessor/controllers/DownloadProjectController.kt

```diff
@@ -548,20 +546,8 @@ class DownloadProjectController(
 
     @Suppress("UnsafeCallOnNullableType")
     private fun getTestResourcesRootAbsolutePath(testRootPath: String,
-                                                 projectRootRelativePath: String): String {
-        // TODO: File should be provided without explicit naming of `save.propeties`
-        val propertiesFile = File(configProperties.repository, projectRootRelativePath)
-            .resolve("$testRootPath/save.properties")
-        val saveProperties: SaveProperties = if (propertiesFile.exists()) {
-            decodeFromPropertiesFile<SaveProperties>(propertiesFile)
```

Comment:
`decodeFromPropertiesFile` is used only here -- maybe remove it?

Reply:
I believe we initially had the following logic: user can provide `save.properties` file with `testFiles` set to custom path. We would then evaluate their properties file and use either custom or default value as `testFiles`. Then we sort of simplified it in save-cli, making `save.properties` and top-level `save.toml` always reside in the same directory.

Reply:
And I believe now we don't need `save.properties` in preprocessor at all